hazelcast-client/src/main/java/com/hazelcast/client/impl/HazelcastClientInstanceImpl.java

```diff
@@ -222,6 +224,8 @@ public HazelcastClientInstanceImpl(ClientConfig config,
         listenerService = initListenerService();
         userContext = new ConcurrentHashMap<String, Object>();
         nearCacheManager = clientExtension.createNearCacheManager();
+        nearCacheContext = new NearCacheContext(serializationService, executionService, nearCacheManager,
+                config.getClassLoader());
```

Comment:
I've added the `ClassLoader` here, since I could not see any negative effect of doing so. We have dealt with a `null` instance here before, so the critical change would be if the `ClassLoader` is actually set.